app/routes/unit_tests/test_projects.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from flask import jsonify, current_app
+from unittest.mock import patch
+from mock_data import MOCK_DATA
+
+class status:
+    HTTP_201_CREATED = 201
+    HTTP_200_OK = 200
+    HTTP_400_BAD_REQUEST = 400
+    HTTP_204_NO_CONTENT = 204
+
+from flytrap import app
+app.config["TESTING"] = True
+
+@pytest.fixture
+def client():
+    return app.test_client()
+
+@patch('app.routes.projects.add_project')
+@patch('app.routes.projects.generate_uuid')
+def test_create_project(mock_generate_uuid, mock_add_project, client):
+    mock_generate_uuid.return_value = "dajhew32876dcx79sd2332"
+    mock_add_project.return_value = MOCK_DATA["create_project"]
```

Comment:
Why is the return value of the route added to the model?

Reply:
Good catch again. There's actually no need for a return value for the model in this case.